src/main/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDecider.java

```diff
@@ -413,7 +426,7 @@ public Decision canAllocate(ShardRouting shardRouting, RoutingNode node, Routing
                             Strings.format1Decimals(freeDiskPercentage, "%"), node.nodeId());
                 }
                 return allocation.decision(Decision.NO, NAME, "less than required [%s%%] free disk on node, free: [%s%%]",
```

Comment:
I believe this message needs to be changed now, since the first value is the "used" value now